dp/LCS_string.py

```diff
@@ -0,0 +1,59 @@
+# def LCSS(str1: str, str2: str, strlen1: int, strlen2: int):
```

Comment:
Python code begins with `#! /usr/bin/env python3`

We should use standard idioms to improve readability.